test/jdk/javax/swing/SwingWorker/TestDoneBeforeDoInBackground.java

```diff
@@ -124,10 +129,10 @@ public void propertyChange(PropertyChangeEvent evt) {
         }
         System.out.println("doInBackground " + doInBackgroundFinished.get() +
                            " getState " + worker.getState());
-        if (doInBackgroundFinished.get() &&
-              worker.getState() != SwingWorker.StateValue.DONE) {
-            throw new RuntimeException("doInBackground is finished " +
-                                       " but State is not DONE");
+        if (worker.getState() != SwingWorker.StateValue.DONE
+            && doInBackgroundFinished.get()) {
+              throw new RuntimeException("doInBackground is finished " +
+                                         "but State is not DONE");
```

Comment:
```suggestion
        if (worker.getState() != SwingWorker.StateValue.DONE
            && doInBackgroundFinished.get()) {
            throw new RuntimeException("doInBackground is finished " +
                                       "but State is not DONE");
```
Throw shouldn't be indented by additional two spaces.

With the `&&` at the start of the line, it is rather clear where condition ends. Additionally, `throw` cannot be part of the condition.

It's a catch with `if` statement because its condition is always indented by four spaces. If required, a blank line could be added in the body of `if` to separate the condition visually; continuation line could use 8-space indentation.